### PR TITLE
Reapply appearance-tools theme support from #3846

### PR DESCRIPTION
**Pull request by tellthemachines**
Trac ticket: https://core.trac.wordpress.org/ticket/60118

I just copied over the changes from #3846 as that patch was no longer applying cleanly.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: tests/phpunit/tests/theme/wpThemeJsonResolver.php [hunk 1]
```diff
@@ -455,15 +458,18 @@ public function test_add_theme_supports_are_loaded_for_themes_without_theme_json
 		);
 		add_theme_support( 'editor-color-palette', $color_palette );
 		add_theme_support( 'custom-line-height' );
+		add_theme_support( 'appearance-tools' );
 
 		$settings = WP_Theme_JSON_Resolver::get_theme_data()->get_settings();
 
 		remove_theme_support( 'custom-line-height' );
 		remove_theme_support( 'editor-color-palette' );
+		add_theme_support( 'appearance-tools' );
```
Comment (andrewserong) [inline tests/phpunit/tests/theme/wpThemeJsonResolver.php]: Should this be `remove_theme_support`?

```suggestion
		remove_theme_support( 'appearance-tools' );
```

Comment (tellthemachines) [reply tests/phpunit/tests/theme/wpThemeJsonResolver.php]: oh whoops yes it should 😅

File: tests/phpunit/tests/theme/wpThemeJsonResolver.php [hunk 2]
```diff
@@ -433,6 +433,9 @@ public function data_get_core_data() {
 
 	/**
 	 * @ticket 54336
+	 * @ticket 60118
+	 *
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/theme/wpThemeJsonResolver.php]:
```suggestion
	 *
```

Fix PHPCS

File: tests/phpunit/tests/theme/wpThemeJsonResolver.php [hunk 3]
```diff
@@ -455,15 +458,18 @@ public function test_add_theme_supports_are_loaded_for_themes_without_theme_json
...
 		$settings = WP_Theme_JSON_Resolver::get_theme_data()->get_settings();
 
 		remove_theme_support( 'custom-line-height' );
 		remove_theme_support( 'editor-color-palette' );
+		remove_theme_support( 'appearance-tools' );
 
 		$this->assertFalse( wp_theme_has_theme_json() );
 		$this->assertTrue( $settings['typography']['lineHeight'] );
 		$this->assertSame( $color_palette, $settings['color']['palette']['theme'] );
+		$this->assertTrue( $settings['border']['color'], 'Support for appearance-tools was not added.' );
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/theme/wpThemeJsonResolver.php]:
```suggestion
		$this->assertTrue( $settings['border']['color'], 'Support for "appearance-tools" was not added.' );
```

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks @tellthemachines for the PR. Left some nit-pick feedbacks.

**Comment by tellthemachines** [comment]
Thanks for the reviews folks! All changes have been made.

**Review by noisysocks — APPROVED** [review]

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by tellthemachines** [comment]
Thanks for the reviews folks! Committed in [r57255](https://core.trac.wordpress.org/changeset/57255).

**Pull request closed without merging**